### ISSUE TITLE
Error launching oha on modern linux (Ubuntu 22.04 or Kali linux)

### ISSUE THREAD

**Issue by rsc1975** [hatoo/oha#188]
Hi, the latest release: v0.5.4 cannot run in some modern Linux, like Ubuntu 22.04 or kali linux, due to libssl version:

I've installed the binary from: https://github.com/hatoo/oha/releases/download/v0.5.4/oha-linux-amd64

Same issue installing using apt tool

```bash
$ oha --version
oha: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

I've searched for this error in stackoverflow and similar places and the solutions are a bit tricky, maybe oha can use a newer version of the lib to avoid it.

My linux uname:

```
Linux ThinkPad-Rob 5.15.57.1-microsoft-standard-WSL2 #1 SMP Wed Jul 27 02:20:31 UTC 2022 x86_64 GNU/Linux
```

```
$ openssl version
OpenSSL 3.0.5 5 Jul 2022 (Library: OpenSSL 3.0.5 5 Jul 2022)
```

Let me know if you need further details

**Comment by hatoo** [comment]
Thanks for the issue!
Yeah, it would be better to build using [rustls](https://github.com/rustls/rustls) instead of [native-tls](https://github.com/sfackler/rust-native-tls) in binary release to avoid this kind of issue.

I'll work this weekend.

**Comment by hatoo** [comment]
I released [v0.5.5](https://github.com/hatoo/oha/releases/tag/v0.5.5) which build with rustls.

Could you check binaries runs on your environment?

**Comment by rsc1975** [comment]
It works like a charm 🎉 

Tested in Ubuntu 22.04 and Kali linux.
